api/app/controllers/spree/api/v2/storefront/cart_controller.rb

```diff
@@ -24,12 +24,26 @@ def add_item
             end
           end
 
+          def remove_item
+            if spree_current_order.nil?
+              render json: { error: "Order doesn't exist" }, status: 422
+            else
+              line_item = spree_current_order.line_items.find(params[:line_item_id])
+
+              spree_authorize! :update, spree_current_order, order_token
+
+              dependencies[:remove_item_from_cart].call(order: spree_current_order, line_item: line_item)
+              render json: serialized_current_order, status: 200
```

Comment:
`status: 200` -> `status: :ok` ?

Reply:
Won't it be better to use 200? You don't have to check which alias means what

Reply:
I think Damian is right, `:ok, :created, etc` are more developer-friendly and human-readable

Reply:
Ok, I will change it 👍